src/components/APIDoc/ApiDoc.tsx

```diff
@@ -35,11 +37,17 @@ export const ApiDoc: React.FunctionComponent<ApiDocProps> = props => {
             </StackItem>
         )}
         { Object.entries(openapi.paths).map(([path, pathObject]) => {
-            return Object.entries(
+            const operations = Object.entries(
                 // Looks like openapi v3.1 supports components here as well
                 pathObject as Record<OpenAPIV3.HttpMethods, OpenAPIV3.OperationObject>
-            ).map(([verb, operation]) =>
-                <StackItem key={`${verb} ${path}`}>
+            )
```

Comment:
I would do something like:
```suggestion
            ).filter(op => op[0].includes(verbs)
```
and continue as before with 
` .map(([verb, operation]) => ...`

We can process the non-operations by separate

Reply:
Updated! Thanks!